config.yaml

```diff
@@ -57,10 +57,6 @@ contracts:
     handler: src/EventHandlers/Gauge.ts
     events:
       - event: NotifyReward(address indexed from, uint256 amount)
-  - name: PriceFetcher
-    handler: src/EventHandlers/PriceFetcher.ts
-    events:
-      - event: PriceFetched(address indexed token, uint256 price)
```

Comment:
Happy to get rid of these. There is more breadth these are providing, but I guess we'll increase the whitelist list size